debian/source/lintian-overrides

```diff
@@ -1,8 +1,2 @@
-# Lintian doesn't see dh-systemd alternative when building on xenial
-ubuntu-advantage-tools: missing-build-dependency-for-dh_-command dh_systemd_start => dh-systemd
-
```

Comment:
Good question. I believe it's OK leaving the override, then.  Although I don't see any of us building the package manually on Xenial, it's still a good idea to document a behaviour that happens there because we still support the distro.